### PR TITLE
change ipfs dag get flag name from format to output-codec

### PR DESCRIPTION
**Pull request by aschmahmann**
Change `ipfs dag get --format` to `ipfs dag get --output-codec` to be closer to https://github.com/ipfs/go-ipfs/pull/8439.

TODO: Merge to master after #8439

### CODE REVIEW

File: core/commands/dag/dag.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ format.
 		cmds.StringArg("ref", true, false, "The object to get").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("format", "f", "Format that the object will be serialized as.").WithDefault("dag-json"),
+		cmds.StringOption("codec", "c", "Format that the object will be encoded as.").WithDefault("dag-json"),
```
Comment (aschmahmann) [inline core/commands/dag/dag.go:112]: "c" is used for the global `--config`. We can either not use a shortcut or come up with a different name/leave things as is.

Comment (lidel) [reply core/commands/dag/dag.go:112]: I don't think short notation adds value here, I'm for just removing it (see https://github.com/ipfs/go-ipfs/pull/8439#discussion_r715076778)

Comment (aschmahmann) [reply core/commands/dag/dag.go:112]: Going to remove the shortcut and change to `--output-codec` to clarify that the stored data is being transcoded per https://github.com/ipfs/go-ipfs/pull/8439#issuecomment-928151167.

File: core/commands/dag/get.go [hunk 2]
```diff
@@ -22,9 +22,9 @@ func dagGet(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) e
 		return err
 	}
 
-	format, _ := req.Options["format"].(string)
-	var fCodec mc.Code
-	if err := fCodec.Set(format); err != nil {
+	codecStr, _ := req.Options["output-codec"].(string)
```
Comment (aschmahmann) [inline core/commands/dag/get.go:25]: This lack of error handling seems wrong, although this pattern is all over the commands lib usage so likely not too big a problem

**Review by guseggert — APPROVED** [review]

**Pull request merged**